Main is broken because of the OAuthClient changes

### Before reporting an issue

- [x] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

testsuite

### Describe the bug

The main branch does not compile right now:

```
[INFO] -------------------------------------------------------------
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/rmartinc/keycloak/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/tokenexchange/StandardTokenExchangeV2Test.java:[95,24] cannot find symbol
  symbol:   class AccessTokenResponse
  location: class org.keycloak.testsuite.util.oauth.OAuthClient
[ERROR] /home/rmartinc/keycloak/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/tokenexchange/StandardTokenExchangeV2Test.java:[107,24] cannot find symbol
  symbol:   class AccessTokenResponse
  location: class org.keycloak.testsuite.util.oauth.OAuthClient
[ERROR] /home/rmartinc/keycloak/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/tokenexchange/StandardTokenExchangeV2Test.java:[120,24] cannot find symbol
  symbol:   class AccessTokenResponse
  location: class org.keycloak.testsuite.util.oauth.OAuthClient
```


### Version

999

### Regression

- [ ] The issue is a regression

### Expected behavior

It should compile.

### Actual behavior

Errors

### How to Reproduce?

```
mvn clean install -DskipTests
```

### Anything else?

_No response_

--- a/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/tokenexchange/StandardTokenExchangeV2Test.java
+++ b/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/tokenexchange/StandardTokenExchangeV2Test.java
@@ -37,7 +37,6 @@ import org.keycloak.testsuite.Assert;
 import org.keycloak.testsuite.arquillian.annotation.EnableFeature;
 import org.keycloak.testsuite.arquillian.annotation.UncaughtServerErrorExpected;
 import org.keycloak.testsuite.util.oauth.AccessTokenResponse;
-import org.keycloak.testsuite.util.oauth.OAuthClient;
 
 import static org.junit.Assert.assertTrue;
 import static org.keycloak.testsuite.auth.page.AuthRealm.TEST;
@@ -92,7 +91,7 @@ public class StandardTokenExchangeV2Test extends AbstractStandardTokenExchangeTe
         oauth.realm(TEST);
         String accessToken = getInitialAccessTokenForClientExchanger();
         {
-            OAuthClient.AccessTokenResponse response = oauth.doTokenExchange(TEST, accessToken, "target", "client-exchanger", "secret");
+            AccessTokenResponse response = oauth.doTokenExchange(TEST, accessToken, "target", "client-exchanger", "secret");
             Assert.assertEquals(OAuth2Constants.REFRESH_TOKEN_TYPE, response.getIssuedTokenType());
             String exchangedTokenString = response.getAccessToken();
             TokenVerifier<AccessToken> verifier = TokenVerifier.create(exchangedTokenString, AccessToken.class);
@@ -104,7 +103,7 @@ public class StandardTokenExchangeV2Test extends AbstractStandardTokenExchangeTe
             assertTrue(exchangedToken.getRealmAccess().isUserInRole("example"));
         }
         {
-            OAuthClient.AccessTokenResponse response = oauth.doTokenExchange(TEST, accessToken, "target", "legal", "secret");
+            AccessTokenResponse response = oauth.doTokenExchange(TEST, accessToken, "target", "legal", "secret");
             Assert.assertEquals(OAuth2Constants.REFRESH_TOKEN_TYPE, response.getIssuedTokenType());
             String exchangedTokenString = response.getAccessToken();
             TokenVerifier<AccessToken> verifier = TokenVerifier.create(exchangedTokenString, AccessToken.class);
@@ -117,7 +116,7 @@ public class StandardTokenExchangeV2Test extends AbstractStandardTokenExchangeTe
         }
         {
             //exchange not allowed due the illegal client is not in the client-exchanger audience
-            OAuthClient.AccessTokenResponse response = oauth.doTokenExchange(TEST, accessToken, "target", "illegal", "secret");
+            AccessTokenResponse response = oauth.doTokenExchange(TEST, accessToken, "target", "illegal", "secret");
             Assert.assertEquals(403, response.getStatusCode());
         }
     }

